substrate/primitives/api/test/tests/ui/missing_versioned_method_multiple_vers.stderr

```diff
@@ -6,3 +6,44 @@ error[E0046]: not all trait items implemented, missing: `test3`
+   |
+
+error[E0308]: mismatched types
+  --> tests/ui/missing_versioned_method_multiple_vers.rs:47:20
+   |
+35 | / sp_api::impl_runtime_apis! {
+36 | |     #[api_version(4)]
+37 | |     impl self::Api<Block> for Runtime {
+38 | |         fn test1() {}
+...  |
+47 | |         fn execute_block(_: Block) {
+   | |                          ^ expected `LazyBlock<Header<u64, BlakeTwo256>, ...>`, found `Block<Header<u64, BlakeTwo256>, ...>`
+...  |
+54 | | }
+   | |_- arguments to this function are incorrect
+   |
+   = note: expected struct `LazyBlock<Header<u64, BlakeTwo256>, ...>`
+              found struct `Block<Header<u64, BlakeTwo256>, ...>`
+note: associated function defined here
+  --> $WORKSPACE/substrate/primitives/api/src/lib.rs
+   |
+   |         fn execute_block(block: Block::LazyBlock);
+   |            ^^^^^^^^^^^^^
+   = note: consider using `--verbose` to print the full type name to the console
+help: call `Into::into` on this expression to convert `sp_runtime::generic::Block<sp_runtime::generic::Header<u64, BlakeTwo256>, UncheckedExtrinsic<CryptoBytes<32, Sr25519PublicTag>, RuntimeCall, CryptoBytes<64, (SignatureTag, Sr25519Tag)>, ((frame_system::extensions::check_nonce::CheckNonce<substrate_test_runtime_client::substrate_test_runtime::Runtime>, frame_system::extensions::check_weight::CheckWeight<substrate_test_runtime_client::substrate_test_runtime::Runtime>), CheckSubstrateCall, frame_metadata_hash_extension::CheckMetadataHash<substrate_test_runtime_client::substrate_test_runtime::Runtime>, frame_system::extensions::weight_reclaim::WeightReclaim<substrate_test_runtime_client::substrate_test_runtime::Runtime>), 16777216>>` into `sp_runtime::generic::LazyBlock<sp_runtime::generic::Header<u64, BlakeTwo256>, UncheckedExtrinsic<CryptoBytes<32, Sr25519PublicTag>, RuntimeCall, CryptoBytes<64, (SignatureTag, Sr25519Tag)>, ((frame_system::extensions::check_nonce::CheckNonce<substrate_test_runtime_client::substrate_test_runtime::Runtime>, frame_system::extensions::check_weight::CheckWeight<substrate_test_runtime_client::substrate_test_runtime::Runtime>), CheckSubstrateCall, frame_metadata_hash_extension::CheckMetadataHash<substrate_test_runtime_client::substrate_test_runtime::Runtime>, frame_system::extensions::weight_reclaim::WeightReclaim<substrate_test_runtime_client::substrate_test_runtime::Runtime>), 16777216>>`
+   |
+47 |         fn execute_block(_.into(): Block) {
+   |                           +++++++
```

Comment:
The test is testing for missing versioned method not an invalid method signature. We could update the ui test to fix this error.

Reply:
Fixed